client/blocks/login/style.scss

```diff
@@ -213,30 +214,76 @@
 	margin-bottom: 16px;
 }
 
-.two-factor-authentication__form-button {
+.two-factor-authentication__info {
+	text-align: center;
+}
+
+.two-factor-authentication__small-print {
+	clear: both;
+	font-size: 11px;
+	padding: 1em 0;
+}
+
+.two-factor-authentication__illustration {
+	position: relative;
+	margin-bottom: 1em;
 	width: 100%;
+	height: 180px;
+	pointer-events: none;
 }
 
-.card.two-factor-authentication__form-action {
-	background: var( --color-white );
+.two-factor-authentication__illustration-device {
+	position: absolute;
+	top: 0;
```

Comment:
We should indent position properties under a position declaration.

Reply:
✅